frontends/common/constantFolding.cpp

```diff
@@ -23,50 +23,49 @@ namespace P4 {
 
 const IR::Expression* DoConstantFolding::getConstant(const IR::Expression* expr) const {
     CHECK_NULL(expr);
-    auto cst = get(constants, expr);
-    if (cst != nullptr)
-        return cst;
-    if (expr->is<IR::Constant>())
+    if (expr->is<IR::Constant>()) {
         return expr;
-    if (expr->is<IR::BoolLiteral>())
+    }
+    if (expr->is<IR::BoolLiteral>()) {
         return expr;
+    }
     if (expr->is<IR::ListExpression>()) {
         auto list = expr->to<IR::ListExpression>();
         for (auto e : list->components)
-            if (getConstant(e) == nullptr)
+            if (getConstant(e) == nullptr) {
```

Comment:
Our style guidelines do not require braces for this case.
Since @ChrisDodd prefers not to squander vertical space, I have avoided using them; they take 1 extra line.